extensions/objects/templates/math-latex-string-editor.directive.ts

```diff
@@ -40,7 +40,7 @@ angular.module('oppia').directive('mathLatexStringEditor', [
           // part of an editable list).
           $scope.$watch('$ctrl.value', function() {
             ctrl.localValue = {
-              label: ctrl.value || ''
+              label: ctrl.value.raw_latex || ''
```

Comment:
Do we need to change the filename?

Reply:
We will add that after we setup the SVG generation